doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1745,6 +1745,8 @@ https://demo.dataverse.org/sitemap.xml is the sitemap URL for the Dataverse Proj
 
 Once the sitemap has been generated and placed in the domain docroot directory, it will become available to the outside callers at <YOUR_SITE_URL>/sitemap/sitemap.xml; it will also be accessible at <YOUR_SITE_URL>/sitemap.xml (via a *pretty-faces* rewrite rule). Some search engines will be able to find it at this default location. Some, **including Google**, need to be **specifically instructed** to retrieve it.
 
+On Dataverse installation with more than 50000 Dataverse collections or datasets, sitemap file name is ``sitemap_index.html``, not ``sitemap.xml``. Be aware in previous steps to use the correct file name corresponding to your installation.
```

Comment:
I'm confused. Why not use the library in all cases? And avoid having two code paths and two XML files?

Reply:
My note wasn't precise enough.

In [sitemap specification](https://www.sitemaps.org/protocol.html#index), in the case where is more than 50000 URLs and/or the sitemap file is larger than 50MB, we have to use sitemap index file. The name of this file isn't clearly indicate, but in example there is indicate `http://www.yoursite.com/sitemap_index.xml` or `https://example.com/public/sitemap_index.xml` in [Google developers documentation](https://developers.google.com/search/docs/crawling-indexing/sitemaps/large-sitemaps?hl=en).

The library `sitemapgen4j` generate `sitemap_index.xml` name file by default in the case of sitemap index. You can see it [here](https://github.com/dfabulich/sitemapgen4j/blob/master/src/main/java/com/redfin/sitemapgenerator/SitemapGenerator.java#L220).

So in all cases the library is used, only the name of the sitemap file is change.

I'll have to clarify this in the documentation :+1: